pkg/instancetype/instancetype_test.go

```diff
@@ -1921,7 +1920,6 @@ var _ = Describe("Instancetype and Preferences", func() {
 				Expect(*vmi.Spec.Domain.Devices.AutoattachInputDevice).To(BeTrue())
 				Expect(vmi.Spec.Domain.Devices.Disks[0].Cache).To(Equal(v1.CacheWriteBack))
 				Expect(vmi.Spec.Domain.Devices.Disks[0].IO).To(Equal(v1.IONative))
-				Expect(*vmi.Spec.Domain.Devices.Disks[0].DedicatedIOThread).To(BeFalse())
```

Comment:
Expect it to be nil now?

Reply:
I've just moved the assertions to the independent test below for easier reading, this test really needs to be broken up.

Reply:
https://github.com/kubevirt/kubevirt/compare/2fbd2f2ed4d527fe94560810519ed1b2fb7f2902..e573f0f868307308b8c0e92883b7ff0f29b6b562